client.go

```diff
@@ -1,20 +1,32 @@
-package redis
+package dpredis
 
 import (
-	"context"
+	"encoding/json"
 	"errors"
-	"github.com/ONSdigital/log.go/log"
+	"fmt"
 	"time"
 
+	. "github.com/ONSdigital/dp-redis/interfaces"
```

Comment:
Try and avoid anonymous imports :) The only place we tend to use them is in tests when importing go-convey, and as a rule I'd say we probably only ever want to use them in tests. For production code, we always want to know exactly what is in this package and what we're having to pull from somewhere else

Reply:
Gone with the above change. 👍